plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2766,6 +2796,13 @@ void producer_plugin::log_failed_transaction(const transaction_id_type& trx_id,
 
 // Called from app thread
 void producer_plugin_impl::switch_to_write_window() {
+   if ( _log.is_enabled( fc::log_level::debug ) ) {
+      auto now = fc::time_point::now();
+      fc_dlog( _log, "Read-only threads #${n}, total time across all threads ${t}us",
+               ("n", _ro_thread_pool_size)
+               ("t", now - _ro_read_window_start_time) );
```

Comment:
This is just the time spent in the read window. That would be good to report as well, but I was expecting this to be the total time spent on all threads executing read-only transactions. If there are 3 threads and 3 read-only transactions that each take 30ms (1 on each thread), then the total time would be 90ms worth of read-only transaction processing, not 30ms.

Reply:
Thanks. I will count the total time of all the threads spending in `chain.push_transaction()`